recipes/r-clustergeneration/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = '1.3.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-clustergeneration
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: clusterGeneration_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/clusterGeneration_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/clusterGeneration/clusterGeneration_{{ version }}.tar.gz
+  sha256: 7c591ad95a8a9d7fb0e4d5d80dfd78f7d6a63cf7d11eb53dd3c98fdfb5b868aa
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-mass
+
+  run:
+    - r-base
+    - r-mass
+
+test:
+  commands:
+    - $R -e "library('clusterGeneration')"  # [not win]
+    - "\"%R%\" -e \"library('clusterGeneration')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=clusterGeneration
+  license: GPL (>= 2)
+  summary: We developed the clusterGeneration package to provide functions  for generating random
+    clusters, generating random  covariance/correlation matrices, calculating a separation
+    index (data and population version) for pairs of clusters or cluster distributions,
+    and 1-D and 2-D projection plots to visualize clusters.  The package also contains
+    a function to generate random clusters based on factorial designs with factors such
+    as degree of separation, number of clusters, number of variables, number of noisy
+    variables.
+  license_family: GPL3
+
```

Comment:
license_file is missing and if you like you can add a few more maintainers.
In general using this script, will help you a little bit: https://github.com/bgruening/conda_r_skeleton_helper

Maintainers are here: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml